source/_docs/pantheon-yml.md

```diff
@@ -1,64 +1,76 @@
 ---
 title: The pantheon.yml Configuration File
-description: Set project configuration in a pantheon.yml file stored in the root of the Drupal or Wordpress site's code repository.  
+description: Set site configuration in a pantheon.yml file stored in the root of their code repository.  
 categories: [developing]
 tags: [code, platform]
 ---
 
-Configuration stored in [YAML](https://en.wikipedia.org/wiki/YAML) files has become a standard across PaaS (Platform as a Service) providers. The `pantheon.yml` file allows developers to hook into platform workflows. It will soon give developers the ability to configure other settings for their sites.
+Hook into platform workflows and add advanced site configuration via the `pantheon.yml` file. To get started, copy [`example.pantheon.yml`](https://github.com/pantheon-systems/quicksilver-examples/blob/master/example.pantheon.yml) to `pantheon.yml` in the root of your site's code repository.
 
-## About pantheon.yml
-The file can contain two collections, `api version:` and `workflows:`. For a template, see [`example.pantheon.yml`](https://github.com/pantheon-systems/quicksilver-examples/blob/master/example.pantheon.yml).   
 
-### API Version
-The current API version for `pantheon.yml` is `1`. This is the only valid value to place in this field. Additional valid values may be introduced in the future.
+## Use pantheon.yml for:
+* [Advanced Site Configuration](#advanced-site-configuration)
+  - [Nested Docroot](#nested-docroot)
+  - [PHP Version](#php-version)
+  - [Protected paths](#protected-paths)
+* [Quicksilver Platform Integration Hooks](#quicksilver-platform-integration-hooks)
+
```

Comment:
Suggest removing this section - seems a little redundant to list the internal hashes here. I think the TOC provides sufficient context for using `pantheon.yml`. Defer to @nataliejeremy on final call